gradebook/src/GradeBook/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+
+
+/* 
+Your your name with the command line arguments as the application assumes the first string as the name of the user
+
+The class Program is the main class encapsulating the main method. 
+
+ */
+namespace GradeBook
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {   
+            var book = new book("first", args);
```

Comment:
First argument to the `book` constructor seems like it should be the user name?

Reply:
Yeah, So this is the name of the gradebook. I'll switch to a be better way. I should probably define a string which clearly tells the purpose of this is the name of gradebook. But to clarify, this is not the username but the name of the gradebook.

Reply:
"first" is now assigned to a var and then the var is passed in the arguments.

Reply:
I think you can make this clearer by making `Book` not care about `args` at all.  I would change the constructor to be `public Book(string gradebookName, string studentName)` and propagate those changes throughout.  A reusable class like this should really care about argument parsing and validation - that should all be taken care of by the calling class.  For instance, you might want to reuse this in a web app, and then the `args` parameter doesn't make sense at all.

Reply:
done